application/app/components/fiches/shared/PotentielSolaireCard.tsx

```diff
@@ -0,0 +1,123 @@
+import { CircleHelp, HousePlug, University, Users, Zap } from 'lucide-react';
+
+const UNKNOWN_TEXTS = {
+	potentiel_solaire: '—',
+	nb_etablissements: '—',
+	nb_eleves: "Nombre d'élèves concernés inconnu",
+};
+
+const SOLAR_TEXT = {
+	high: 'Le potentiel solaire de cet établissement me paraît tout à fait rayonnant',
+	low: 'Des optimisations sont à prévoir pour un bon potentiel solaire',
+};
+
+const FOYER_CONSO_KWH = 2300;
+const PERSONNES_PAR_FOYER = 2;
+const HIGH_SOLAR_THRESHOLD = 500_000;
+
+interface PotentielSolaireCardProps {
+	potentiel_solaire?: number;
+	nb_etablissements?: number;
+	niveau?: string;
+	nb_eleves?: number;
+	showInterpretation?: boolean;
+	showNbEtablissements?: boolean;
+}
```

Comment:
Tu peux utiliser un type pour les niveaux et apres gerer le texte du niveau avec une fonction 

```suggestion
type EtablissementNiveau = 'lycee' | 'college' | 'primaire';

interface PotentielSolaireCardProps {
	potentiel_solaire?: number;
	nb_etablissements?: number;
	niveau?: EtablissementNiveau;
	nb_eleves?: number;
	showInterpretation?: boolean;
	showNbEtablissements?: boolean;
}

function getNiveauLabel(niveau?: EtablissementNiveau) {
	if (niveau === undefined) return '-';
	if (niveau === 'college') return 'de colleges';
	if (niveau === 'primaire') return 'blabla';
	if (niveau === 'lycee') return 'blBLA';

	throw new Error('The niveau given doesnt exist.');
}
```